docs/build-with-rust.md

```diff
@@ -0,0 +1,354 @@
+## Build an IoT App with Go
+
+### Architecture
+
+In this section, we will walk through and explain the code for the different commands.
+As explained in the Getting Started page, the project is structured as follow:
+
+-   migrate (`/bin/migrate/main.rs`) - creates the `carepet` keyspace and tables
+-   collar (`/bin/sensor/main.rs`) - generates a pet health data and pushes it into the storage
+-   web app (`/main.rs`) - REST API service for tracking pets health state
+
+### Migrate
+
+The below command allows you to get node IP address:
```

Comment:
This assumes we already have a running Scylla Docker, but I could not find where are the instruction to run it

Reply:
Look getting_started.md the instructions but they are for the Go version
We should either:
- make the getting_started.md generic and have it as a prerequisite
- have a getting start per lang

Reply:
Great suggestion. I'd suggest the following:
1. Have the `getting_started.md` to start the database (up to `docker exec -it carepet-scylla1 nodetool status`)
2. Add links to respective languages and add all the steps even if it's repetitive